descwl_shear_sims/tests/test_gen_sip_wcs.py

```diff
@@ -0,0 +1,139 @@
+import os
+import pytest
+import numpy as np
+import galsim
+
+from ..gen_sip_wcs import gen_sip_wcs
+
+
+@pytest.mark.skipif(
+    "CATSIM_DIR" not in os.environ,
+    reason='simulation input data is not present')
+def test_gen_sip_wcs_smoke():
+    seed = 42
+    world_origin = galsim.CelestialCoord(
+        ra=20 * galsim.degrees,
+        dec=-10 * galsim.degrees)
+    origin = galsim.PositionD(x=10, y=11)
+    gen_sip_wcs(
+        rng=np.random.RandomState(seed=seed),
+        position_angle_range=(0, 360),
+        dither_range=(-0.5, 0.5),
+        scale=0.25,
+        scale_frac_std=0.1,
+        shear_std=0.1,
+        world_origin=world_origin,
+        origin=origin)
+
+
+@pytest.mark.skipif(
+    "CATSIM_DIR" not in os.environ,
+    reason='simulation input data is not present')
```

Comment:
These tests don't appear to depend on the catsim dir

Reply:
I am not sure they are running.

Reply:
the sip version relies on data in the catsim dir

Reply:
ahh I see it now. We need this to be controlled by the input RNG eventually.